src/runtime_src/core/include/ert.h

```diff
@@ -413,6 +413,18 @@ enum softkernel_type {
   SOFTKERNEL_TYPE_XCLBIN = 1,
 };
 
+/*
+ * Base address GPIO per spec
+ * | Offset  | Description
+ * -----------------------
+ * | 0x00    | ERT_MGMT_PF_base_addr (Not sure where this should be use)
+ * | 0x08    | ERT_USER_PF_base_addr. The base address of ERT peripherals
+ */
+#if defined(ERT_BUILD_V20)
+uint32_t ert_base_addr;
```

Comment:
initialize to 0

Reply:
Compiler would initial global variable. But I would initial it explicitly here.